app/controllers/api/v1/books/rents_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    module Books
+      class RentsController < ApiController
+        def index
+          render_paginated Rent.where(book_id: params[:book_id])
```

Comment:
In index endpoints it is important to take into account the problem of the N+1 queries.
http://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations
http://tomdallimore.com/blog/includes-vs-joins-in-rails-when-and-where/

See that you are serializing the user and the book for every rent... could there be a better way to make the queries here??? 🤔 🤔 🤔